apps/easypid/src/features/receive/FunkeOpenIdCredentialNotificationScreen.tsx

```diff
@@ -463,19 +334,19 @@ export function FunkeCredentialNotificationScreen() {
               ),
             }
           : undefined,
-        isAuthFlow && shouldUsePinForPresentation
+        resolvedOffer.flow === 'auth-presentation-during-issuance' && shouldUsePinForPresentation
           ? {
               step: 'pin-enter',
               progress: 82.5,
               screen: <PinSlide key="pin-enter" isLoading={isSharingPresentation} onPinSubmit={onPresentationAccept} />,
             }
           : undefined,
-        isPreAuthWithTxFlow
+        resolvedOffer.flow === 'pre-auth' && resolvedOffer.txCodeInfo
```

Comment:
Could also add a flow of `pre-auth-with-transaction` as the UI is also quite different there.